recipes-soc/rdk-gstreamer-utils-platform/rdk-gstreamer-utils-platform.bb

```diff
@@ -0,0 +1,28 @@
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${MANIFEST_PATH_PLATFORM}/LICENSE;md5=5bfad6e034e497ee148eec56e175c6e8"
```

Comment:
Usually the checksum would refer to the LICENSE file in SRC_URI - is this possible?

Reply:
Hi @sajanasadhan99 ,
please see https://github.com/rdkcentral/meta-vendor-raspberrypi-dev/blob/develop/recipes-soc/sysint-soc/sysint-soc.bb#L4 which is doing the same.

Reply:
Done